From b9fed3d04d5e8cff6f0ad318cc9ab07831af78c0 Mon Sep 17 00:00:00 2001
From: Daniel Svensson <dsvensson@gmail.com>
Date: Wed, 1 Apr 2015 09:38:13 +0200
Subject: [PATCH] test_rollsum needs -lz to link.

---
 Makefile-tests.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile-tests.am b/Makefile-tests.am
index 675547d..fa35588 100644
--- a/Makefile-tests.am
+++ b/Makefile-tests.am
@@ -119,7 +119,7 @@ TESTS_LDADD = $(ostree_bin_shared_ldadd) $(OT_INTERNAL_GIO_UNIX_LIBS)
 
 tests_test_rollsum_SOURCES = src/libostree/ostree-rollsum.c tests/test-rollsum.c
 tests_test_rollsum_CFLAGS = $(TESTS_CFLAGS)
-tests_test_rollsum_LDADD = libbupsplit.la $(TESTS_LDADD)
+tests_test_rollsum_LDADD = libbupsplit.la $(TESTS_LDADD) $(OT_DEP_ZLIB_LIBS)
 
 tests_test_mutable_tree_CFLAGS = $(TESTS_CFLAGS)
 tests_test_mutable_tree_LDADD = $(TESTS_LDADD)
