From: David King <dking@redhat.com>
Date: Thu, 2 Apr 2015 13:02:39 +0200
Subject: [PATCH] GPG test needs gpgme to link

--- a/Makefile-tests.am
+++ b/Makefile-tests.am
@@ -147,7 +147,7 @@ TESTS_LDADD = $(ostree_bin_shared_ldadd) $(OT_INTERNAL_GIO_UNIX_LIBS)
 	src/libostree/ostree-gpg-verify-result-private.h \
 	tests/test-gpg-verify-result.c
 tests_test_gpg_verify_result_CFLAGS = $(TESTS_CFLAGS)
-tests_test_gpg_verify_result_LDADD = $(TESTS_LDADD)
+tests_test_gpg_verify_result_LDADD = $(TESTS_LDADD) $(GPGME_LIBS)
 
 EXTRA_DIST += \
 	tests/gpg-verify-data/README.md \
