From: Simon McVittie <smcv@debian.org>
Date: Fri, 11 Mar 2016 08:46:27 +0000
Subject: test-libarchive: fix underlinking

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 Makefile-tests.am | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile-tests.am b/Makefile-tests.am
index 6ec5835..4b0a526 100644
--- a/Makefile-tests.am
+++ b/Makefile-tests.am
@@ -179,8 +179,8 @@ tests_test_checksum_CFLAGS = $(TESTS_CFLAGS) $(libglnx_cflags)
 tests_test_checksum_LDADD = $(TESTS_LDADD)
 
 tests_test_libarchive_import_SOURCES = tests/test-libarchive-import.c
-tests_test_libarchive_import_CFLAGS = $(TESTS_CFLAGS) $(libglnx_cflags)
-tests_test_libarchive_import_LDADD = $(TESTS_LDADD)
+tests_test_libarchive_import_CFLAGS = $(TESTS_CFLAGS) $(libglnx_cflags) $(OT_DEP_LIBARCHIVE_CFLAGS)
+tests_test_libarchive_import_LDADD = $(TESTS_LDADD) $(OT_DEP_LIBARCHIVE_LIBS)
 
 tests_test_keyfile_utils_CFLAGS = $(TESTS_CFLAGS)
 tests_test_keyfile_utils_LDADD = $(TESTS_LDADD)
