From: Simon McVittie <smcv@collabora.com>
Date: Tue, 29 Aug 2017 17:01:41 +0100
Subject: test-pull-bareuser.sh: This test uses bare-user, hence needs xattrs

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 tests/test-pull-bareuser.sh | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tests/test-pull-bareuser.sh b/tests/test-pull-bareuser.sh
index 0729247..2e35723 100755
--- a/tests/test-pull-bareuser.sh
+++ b/tests/test-pull-bareuser.sh
@@ -21,6 +21,7 @@ set -euo pipefail
 
 . $(dirname $0)/libtest.sh
 
+skip_without_user_xattrs
 setup_fake_remote_repo1 "archive"
 
 repo_mode=bare-user
