From: Simon McVittie <smcv@debian.org>
Date: Sun, 26 Jun 2016 13:57:13 +0100
Subject: tests: fail the build if symlinking tests/ostree fails

Signed-off-by: Simon McVittie <smcv@debian.org>
---
 Makefile-tests.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile-tests.am b/Makefile-tests.am
index 25b8202..9be9061 100644
--- a/Makefile-tests.am
+++ b/Makefile-tests.am
@@ -245,7 +245,8 @@ ALL_LOCAL_RULES += tests/libreaddir-rand.so
 CLEANFILES += tests/libreaddir-rand.so tests/ostree-symlink-stamp tests/ostree
 
 tests/ostree-symlink-stamp: Makefile
-	@real_bin=`cd $(top_builddir) && ./libtool --mode=execute echo ostree`; \
+	@set -e; \
+	real_bin=`cd $(top_builddir) && ./libtool --mode=execute echo ostree`; \
 	ln -sf "$${real_bin}" tests/ostree; \
 	touch $@
 
